extensions/Worker.Extensions.Shared/Serialization/JsonSerializerProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Text.Json;
+internal static class JsonSerializerOptionsProvider
+{
+    // Note: If you need an instance with different settings, create a new instance and use that
+    // instead of modifying this as it may break other places where this has been used.
+
+    /// <summary>
+    /// Option with the below settings:
+    ///  1. Deserialization supports case-insensitive properties.
+    ///  2. Serialization produces camelCase properties.
+    /// </summary>
+    internal static readonly JsonSerializerOptions Options = new JsonSerializerOptions
```

Comment:
The heart of this issue is that the timer converter wants to control its own serialization behavior. As such, I don't think it is worth having this in a shared area. I would prefer the `TimerPocoConverter` had this directly as a field.

Reply:
Moved to timer extension project.